app/forms/concerns/decidim/extra_user_fields/forms_definitions.rb

```diff
@@ -76,11 +86,38 @@ def location?
         extra_user_fields_enabled && current_organization.activated_extra_field?(:location)
       end
 
+      def underage?
+        extra_user_fields_enabled && current_organization.activated_extra_field?(:underage)
+      end
+
+      def underage_accepted?
+        underage? && underage == "1"
+      end
       # EndBlock
 
       def extra_user_fields_enabled
         @extra_user_fields_enabled ||= current_organization.extra_user_fields_enabled?
       end
+
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
+
+      def birth_date_under_limit
```

Comment:
We should split this method into small ones. 

I wonder if we should move birth_date validations in a dedicated module to include but for now it should be goot